enhancements/machine-api/aws-efa-networking.md

```diff
@@ -181,9 +246,34 @@ In the first phase, we are planning to just provide users the ability to create
 We will run standard regression testing on the feature but we do not expect any special additional testing to be
 performed.
 
-In the future, if/when we introduce full support for this feature, we will need to introduce specific testing to check
-that the workloads can leverage the EFA feature and that their networking throughput is increased when leveraging the
-feature.
+There are two levels of testing that we can then automate within the origin test suite.
+
+#### Smoke test for basic networking
+
+To ensure that enabling an EFA interface does not break basic networking requirements,
+we will add a test performs the following steps:
+
+- Create a new MachineSet based on the existing MachineSets within the cluster
+  - Modify the MachineSet to enable EFA networking
+  - Set the MachineSet replicas to 1
+- Wait for the MachineSet to create a Machine
+- Wait for the Machine to become a Node
+- Remove the MachineSet once the Machine has a Node
+- Wait for the Machine to be removed
+
+This process will test whether the basic Node networking is affected by the introduction of the EFA adapter.
+As the EFA adapter requires a specific kernel module to provide basic networking, this will also provide signal as to
+the inclusion of the `efa` kernel module with the RHCOS image.
+
+#### Extended MPI testing
```

Comment:
I don't care whether this is automated or done manually by QE, but I see this as the minimum bar to claim the feature works and it must be done every release to ensure it keeps working.

Reply:
Of course